shared/offline_compiler/source/CMakeLists.txt

```diff
@@ -4,9 +4,8 @@
 # SPDX-License-Identifier: MIT
 #
 
-project(${OCLOC_NAME}_lib)
+project(ocloc_lib)
```

Comment:
why is this change needed? I see what can be incorrect with our current implementation - project is defined before OCLOC_NAME is defined which evaluates to `project(_lib)` . Simply moving `set(OCLOC_NAME "ocloc")` to a line before `project(${OCLOC_NAME}_lib)` should fix this issue

Reply:
It's not really needed. It just looked more consistent with the rest of this patch.